functest/debug_test.go

```diff
@@ -61,7 +61,7 @@ func TestDebugSimple(t *testing.T) {
 		runChannel := make(chan error)
 		containerName := "testDebugSimpleContainer" + strings.ReplaceAll(stackRaw[i], "/", "_")
 		go func() {
-			_, err := cmdtest.RunAppsody(sandbox, "debug", "--name", containerName)
+			_, err := cmdtest.RunAppsody(sandbox, "debug", "--name", containerName, "-P")
```

Comment:
We could look further into which tests actually use ports... We don't have tests which check that the ports specified in the Dockefile-stack are actually getting used...

Reply:
Yeah this could definitely be something to look into as a different PR, there is an underlying issue that still intermittently fails which could likely be caused by misallocation of ports.